src/domain/app/layout/__tests__/__snapshots__/PageLayout.test.tsx.snap

```diff
@@ -651,7 +651,7 @@ exports[`PageLayout matches snapshot 1`] = `
               © 
               Copyright
                
-              2024
+              2025
```

Comment:
The test would be good to fix in a long term way so that this test wouldn't break come 2026. Not necessarily in this PR though.